src/bin.ts

```diff
@@ -129,6 +134,8 @@ const service = register({
   project,
   preferTsExts,
   logError,
+  scriptMode,
+  fileToExecute,
```

Comment:
Instead of adding a new option with conditions, why not just make `-s` behave like `cwd: dirname(fileToExecute)`?

Reply:
I see what you mean, I'll change this